cmd/snap/complete.go

```diff
@@ -120,46 +212,73 @@ func (cps connectPlugSpec) Complete(match string) []flags.Completion {
 
 	var ret []flags.Completion
 
-	var snapPrefix, plugPrefix string
+	var prefix string
 	if len(parts) == 2 {
 		// The user typed the colon, means they know the snap they want;
 		// go with that.
```

Comment:
We also support colon followed by slot (i.e. snap name is empty), in which case we default to core, e.g.
`snap connect foosnap:home :home`
`snap disconnect :home`

It's a bit hard to see if that's supported by the code below? See ResolveConnect / ResolveDisconnect in the repo.